IPCheck/dashAnalysis.sh

```diff
@@ -113,7 +113,8 @@ elif [ $KIND_OF = "Nodes" ]; then
 	echo ": DASH Analysis > " $ANALYSIS_RESULT_FILE
 	echo ": DASH Analysis > " $ANALYSIS_RESULT_FILE >> $LOG_FILE
 	java -jar $DASH_LIB $JSON_FILE -summary $ANALYSIS_RESULT_FILE >> $LOG_FILE
-	cp $SRC_DIR/package-lock.json $DEST_DIR
+	cp $SRC_DIR/package.json $DEST_DIR
+	cp $SRC_DIR/yarn.lock $DEST_DIR
```

Comment:
I think, `yarn.lock` will only exist if `yarn` is used as package manager for a nodejs project. If `npm` is used (which is the default, I believe) only `package-lock.json` might exist, and this command will fail.

It might be good to test for the existence of `yarn.lock` and `package-lock.json` and only copy the file that is found.